docs/pages/desktop-access/discovery.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Desktop Discover
+description: Automatically add Teleport Desktop Access to your infrastructure
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="11"
+>
+
+The features documented on this page are available in Teleport `11` and higher.
+
+</Details>
+
+This page docments the Teleport Desktop Access Discovery wizard. This feature, available through the Web UI, makes it easy to create a Windows Active Directory and configure it to accept Desktop Access connections from Teleport.
+
+## Prerequisites
+
+- A server or virtual machine running a Windows Server operating system.
+- A host where you will run the Teleport Desktop Service.
+- <ScopedBlock scope={["oss", "enterprise"]}> This **can not** be the same instance
+  of Teleport running your Auth and/or Proxy service(s). To install Desktop Access into an existing Teleport instance running other services, see the [Getting Started](getting-started.mdx) guide for Desktop Access.</ScopedBlock>
+
+  See [Installation](../../installation.mdx) for details.
+
+## Step 1/2. Install Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/2. Run the discovery wizard
+
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>`mytennant.teleport.sh`</ScopedBlock>. Select "Desktops" from the main menu, then **ADD DESKTOP**:
+
+![ADD DESKTOP in the Teleport web UI](../../img/desktop-access/add-desktop.png)
+
+This will bring you to the "Add New Resorces" menu. "Desktop" will already be selected. Click **NEXT**.
+
+If your Windows server is already running as an Active Directory controller, skip to the next step. Otherwise, copy and paste the first command provided into a Windows Powershell window. If your server is not already running AD Certificate Services, copy and paste the second command after the first one completes and the server restarts:
```

Comment:
```suggestion
Teleport provides three commands that you can run to perform starts of the set up process. Which commands you need to run depends on what is already configured.

1. Install Active Directory. Skip this step if Active Directory is already running.
2. Install AD Certificate Services. Skip this step if AD CS is already running.
3. Configure AD for Teleport. This step should always be run.
```

In most real world deployments, the same server is not used as a DC, for AD CS, and for regular access. Suggested rephrasing this to be a bit more generic.